fiftyone/core/clips.py

```diff
@@ -275,12 +275,12 @@ def reload(self):
         #
         # Regenerate the clips dataset
         #
-        # This assumes that calling `load_view()` when the current clips
+        # This assumes that calling `load_saved_view()` when the current clips
         # dataset has been deleted will cause a new one to be generated
         #
 
         self._clips_dataset.delete()
-        _view = self._clips_stage.load_view(self._source_collection)
+        _view = self._clips_stage.load_saved_view(self._source_collection)
```

Comment:
This should not have been renamed; `ViewStage.load_view()` is something different